src/docfx/lib/moniker/EvaluatorWithMonikersVisitor.cs

```diff
@@ -66,7 +71,7 @@ public IEnumerable<string> Visit(LogicExpression expression)
         private static Dictionary<string, MonikerProductInfo> InitializeMonikers(IEnumerable<Moniker> monikers)
         {
             var monikerNameList = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
-            var productNameDictionary = new Dictionary<string, List<string>>(StringComparer.OrdinalIgnoreCase);
+            var productNameDictionary = new SortedDictionary<string, List<Moniker>>(StringComparer.OrdinalIgnoreCase);
```

Comment:
why SortedDictionary

Reply:
I want the stable evaluation output of monikerRange which contains multiple products like `>netcore-1.0 || < netframework-2.0`